src/App/ActivityList.purs

```diff
@@ -29,37 +30,43 @@ type Todo =
   }
 
 type TodoNow = {
-    todos :: Maybe (List Todo) 
+    todos :: Maybe (Array Todo) 
```

Comment:
+1 on your suggestions to make this just `Array Todo` 

When we add the drag and drop feature between panels, `TodoNow` will be triggered by the `select` action.  For now will accept.